pom.xml

```
@@ -30,7 +30,7 @@
   <groupId>org.apereo.uportal</groupId>
   <artifactId>uportal-app-framework</artifactId>
   <packaging>war</packaging>
-  <version>12.2.1-SNAPSHOT</version>
+  <version>12.3.0</version>
```

Ditto comment above that bumping version number in the feature PR is not current process and is going to make [status quo documentation](https://github.com/uPortal-Project/uportal-app-framework/blob/master/docs/releasing.md#java-release) not quite applicable and is going to break status quo automation.

Leave this the SNAPSHOT and when this merges Jenkins will build and deploy a new SNAPSHOT which will trigger the chain that results in a uPortal-home based on this snapshot deployed as predev.

```suggestion
  <version>12.2.1-SNAPSHOT</version>
```